pages/courses/[id].js

```diff
@@ -19,6 +19,7 @@ import { getAllCohorts, getCurrentCohort } from '../../lib/cohorts'
 import { getLessonsSubmissions } from '../../lib/lessons'
 import Image from 'next/image'
 import Loading from '../../components/Loading'
+import {dateFormat} from '../../lib/dateFormat'
```

Comment:
Style fixes: add space between brackets:
```js
import { dateFormat } 
````

Just an opinion and we can discuss it, but I think its good because then we can implement code styling and linter action and testing to automatize PRs.